commandline/src/main/java/tech/pegasys/web3signer/commandline/config/PicoKeystoresParameters.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline.config;
+
+import tech.pegasys.web3signer.commandline.DefaultCommandValues;
+import tech.pegasys.web3signer.signing.config.KeystoresParameters;
+
+import java.nio.file.Path;
+
+import picocli.CommandLine.Option;
+
+public class PicoKeystoresParameters implements KeystoresParameters {
+
+  @Option(
+      names = {"--keystores-path"},
+      description = "The path to a directory storing Eth2 keystores",
+      paramLabel = DefaultCommandValues.MANDATORY_PATH_FORMAT_HELP)
+  private Path keystoresPath;
+
+  @Option(
+      names = {"--keystores-passwords-path"},
+      description =
+          "The path to a directory with the corresponding passwords file for the keystores."
+              + " Filename for the password without the extension must match the keystore filename."
+              + " This cannot be set if --keystores-password-file is also specified.",
```

Comment:
There's no mention of the files needing to end in txt

Reply:
I guess the same goes for the other options too (presumably json is requirement for keystores-path?)

Reply:
Good point. Keystore files must have a json extension and the password files a txt extension. However I wonder how important that restriction is? If we ensure that the directories were different which is a sensible security measure then the file extensions don't matter at all.

Reply:
True. 

Couple of concerns with loading any file extension, if that's what you're suggesting...

1. Any non-keystore files will cause errors and be ignored which is fine, but potentially a lot of wasted processing.

2. What happens if there are duplicates files with only differing extensions? I think this could be a problem, especially in the password directory.

Reply:
Could find a way of dealing with that. But I prefer the simplicity of using .json extension for the keystore and a .txt extension for the password. Added those details in the CLI docs.